src/ecdsa/ot_based_ecdsa/test.rs

```diff
@@ -35,6 +51,57 @@ pub fn run_sign(
     )
 }
 
+type SigWithRerand = (Tweak, Vec<(Participant, Signature)>);
+/// Runs signing by calling the generic run_sign function from crate::test
+/// This signing mimics what should happen in real world, i.e.,
+/// rerandomizing the presignatures
+pub fn run_sign_with_rerandomization(
+    participants_presign: Vec<(Participant, PresignOutput)>,
```

Comment:
Taking ownership here seems redundant
```suggestion
    participants_presign: &[(Participant, PresignOutput)],
```

Reply:
I see this is already and established pattern in the old function. Feel free to update or skip. Not super important.